README.md

```diff
@@ -3,53 +3,67 @@
 
-### Using Dev Tools
+### Use Dev Tools!
 
-Dev Tools should be used to **fix minor issues** on a web page
-*(like dark icon on a dark background, removing bright background, adding a white background to transparent image, etc.)*.
-If the page looks partially dark and bright in **Dynamic mode**, it should be considered as a bug.
-For **Filter mode** it is a common practice to invert already dark page parts.
+Dev Tools is designed to **fix minor issues** on a web page
+*(like dark icon on dark background, removing bright background, adding white background to transparent image, etc.)*.
+If the page looks partially dark and bright in **Dynamic mode**, we consider it a bug.
+For **Filter mode**, it is a common practice to invert already dark page parts.
 
-- Open **Chrome Dev Tools** (F12).
-- Click on **element picker** (top-left corner).
+- Open **Chrome Dev Tools** (F12) in Chrome or "Inspector" (Ctrl+Shift+C) in Firefox.
+- Click on **element picker** (top-left corner). It is enabled automatically in Firefox.
 - Pick wrongly inverted element.
-- Choose a **[selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors)** for that element or all similar elements (e.g. if element has `class="icon small"` selector may look like `.icon`).
+- Choose a **[selector](https://developer.mozilla.org/docs/Web/CSS/CSS_Selectors)** for that element or all similar elements (e.g. if element has `class="icon small"`, selector may look like `.icon`).
 - Click **Dark Reader icon**.
-- Click **Open developer tools** (at bottom).
-- Find or add a block containing URL and selectors to invert.
+- Click **Open developer tools** (at bottom of Dark Reader popup).
+- Edit or add a block containing the URL and selectors to invert.
+- Click **Apply**.
+- Check how it looks both in **Light** and **Dark** mode.
+- If the **fix works** open
+**[dynamic-theme-fixes.config](https://github.com/alexanderby/darkreader/blob/master/src/config/dynamic-theme-fixes.config) file**
+or **[inversion-fixes.config](https://github.com/alexanderby/darkreader/blob/master/src/config/inversion-fixes.config) file**.
+- Click **Edit** (login to GitHub first).
+- **Insert your fix** there. Preserve **alphabetic order** by URL.
+- Provide a **short description** of what you have done.
+- Click **Propose file change**.
+- Review your changes. Click **Create pull request**.
+- The Travis CI will run tests reviewing your changes.
+- If you see a **red cross** click **Details** to see what is wrong and edit the existing Pull Request.
+- When you see a **green checkmark** then everything is fine.
+- Dark Reader developer will **review** and merge your changes, making them available to all users.
+
 ```
 dynamic-theme-fixes.config
-================================
+==========================
```

Comment:
Revert this

Reply:
No problem... just curious, why?  To me, it looks cleaner with the equal signs equaling the header length.

Reply:
It's the convention in our code to have this exact amount of equal signs.

Reply:
Got it. Thanks. I'll revert all changes modifying the number of equal signs.